### PR TITLE
Trim Static Analysis Baselines

### PR DESCRIPTION
**Pull request by AlexandruGG**
This PR removes the Psalm baseline and the PHPStan Docs baseline, which is not in use anymore because the `docs` folder is ignored.

### CODE REVIEW

File: src/Iterator/RandomIterator.php [hunk 1]
```diff
@@ -79,9 +79,10 @@ public function key()
 
     public function next(): void
     {
-        unset($this->indexes[$this->key]);
-
-        $this->key = key($this->indexes);
+        if (null !== $this->key) {
+            unset($this->indexes[$this->key]);
+            $this->key = key($this->indexes);
+        }
```
Comment (AlexandruGG) [inline src/Iterator/RandomIterator.php:85]: @drupol let me know if this looks right to you. If the key is _null_ then the `unset` will not have any effect anyway

Comment (drupol) [reply src/Iterator/RandomIterator.php:85]: Nice catch :)

Comment (AlexandruGG) [reply src/Iterator/RandomIterator.php:85]: Psalm's catch 😄

**Comment by AlexandruGG** [comment]
thanks!

**Comment by drupol** [comment]
Thanks mate!

**Pull request merged**